tests/integration/targets/mso_schema_site_anp_epg_bulk_staticport/tasks/main.yml

```diff
@@ -0,0 +1,580 @@
+# Test code for the MSO modules
```

Comment:
can you see what happens if you use the old module which can add only one static port first and the new bulk module by adding only one static port to the list? Something like below?
```
- name: Add a new static port to a site EPG
  cisco.mso.mso_schema_site_anp_epg_staticport:
    host: mso_host
    username: admin
    password: SomeSecretPassword
    schema: Schema1
    site: Site1
    template: Template1
    anp: ANP1
    epg: EPG1
    type: port
    pod: pod-1
    leaf: 101
    path: eth1/1
    vlan: 126
    deployment_immediacy: immediate
    state: present
  delegate_to: localhost
  
  ```
  ```
  - name: Add static port 1 to site EPG1 of AP1 (check mode)
  mso_schema_site_anp_epg_bulk_staticport: 
    <<: *mso_info
    schema: '{{ mso_schema | default("ansible_test") }}'
    site: '{{ mso_site | default("ansible_test") }}'
    template: Template 1
    anp: AP1
    epg: EPG1
    pod: pod-1
    leaf: 101
    path: eth1/1
    vlan: 126
    mode: native
    type: port
    deployment_immediacy: immediate
    static_ports:
      - path: eth1/1
        leaf: 101
    state: present

Reply:
Doing this will remove the old payload and recreate the whole payload with the new task.